models/points.go

```diff
@@ -741,6 +741,16 @@ func scanLine(buf []byte, i int) (int, []byte) {
 	start := i
 	quoted := false
 	fields := false
+	bufLength := len(buf)
+
+	for {
+		if bufLength > start && buf[start] == 0 {
```

Comment:
I think it is possible if someone purposely do it programmatically.

This fixes is more like a prevention for a programmer's mistake, eg, buffer helps us to generate empty string without we knowing it, and it is not visible until the result from ```len(key)```.is called.